.github/workflows/next.yml

```diff
@@ -22,15 +22,21 @@ jobs:
           - "10.0.x"
           - "10.1.x"
           - "10.2.x"
+          - "10.3.x"
+          - "11.0.x"
         exclude:
           - drupal: "10.0.x"
             php: "8.3"
           - drupal: "10.1.x"
             php: "8.3"
+          - drupal: "11.0.x"
+            php: "8.1"
+          - drupal: "11.0.x"
+            php: "8.2"
```

Comment:
11.0 required PHP 8.3

Reply:
@yobottehg Correct!!
Pipeline is running on 8.3
If configuration is to exclude running with 8.1, 8.2 versions of php for Drupal 11